kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDataPanel.js

```diff
@@ -90,7 +92,7 @@ define([
         options: {
             title: 'Data',
             loadingImage: Config.get('loading_gif'),
-            notLoggedInMsg: "Please log in to view a workspace.",
+            notLoggedInMsg: 'Please log in to view a workspace.',
```

Comment:
This will likely never be seen, but it's a good time to fix it. Should say "Please log in to view Narrative data."